config/ftbquests/quests/chapters/storage.snbt

```diff
@@ -1166,5 +1166,373 @@
+				""
+				"This is meant to be a start, not exhaustive, so check out the quest chapters for each mod for more information."
+			]
+			dependencies: ["00000000000003FF"]
+			id: "568DC3ECDB943C47"
+			tasks: [{
+				id: "5DF4DF16758CC1FC"
+				type: "item"
+				item: "minecraft:bucket"
+			}]
+			rewards: [{
+				id: "4ADE1E2D7CB232BF"
+				type: "xp"
+				xp: 100
+			}]
+		}
+		{
+			title: "Fluid Tank (Mekanism)"
+			x: 7.5d
+			y: -4.5d
+			description: ["Mekanism fluid tanks have a bucket mode (default: shift + scroll), which lets them pick up and use fluids. Try it as a gigantic bucket!"]
+			dependencies: ["568DC3ECDB943C47"]
+			id: "120DA7809C60693C"
+			tasks: [{
+				id: "72362A5E247C68DE"
+				type: "item"
+				item: "mekanism:basic_fluid_tank"
+			}]
+			rewards: [{
+				id: "30EC70E6E72E71F0"
+				type: "xp"
+				xp: 100
+			}]
+		}
+		{
+			title: "Fluid Tank (PNC)"
+			x: 8.5d
+			y: -4.5d
+			description: ["Using a wrench, PneumaticCraft's tanks can be connected with tanks above and below to combine their capacities."]
+			dependencies: ["568DC3ECDB943C47"]
+			id: "42FE114176EAE0DD"
+			tasks: [{
+				id: "61F77FEB2B9801A7"
+				type: "item"
+				item: "pneumaticcraft:small_tank"
+			}]
+			rewards: [{
+				id: "4EDA302F870DAEE1"
+				type: "xp"
+				xp: 100
+			}]
+		}
+		{
+			title: "Fluid Cell"
+			x: 9.5d
+			y: -4.0d
+			description: ["Thermal Series' fluid cells are pretty configurable. Their sides can be configured to automatically input and output into adjacent inventories."]
+			dependencies: ["568DC3ECDB943C47"]
+			id: "5BEB6955518864A8"
+			tasks: [{
+				id: "49D84038D947694F"
+				type: "item"
+				item: "thermal:fluid_cell"
+			}]
+			rewards: [{
+				id: "2B1E9DE51DAB8B6E"
+				type: "xp"
+				xp: 100
+			}]
+		}
+		{
+			title: "Black Hole Tank"
+			x: 8.5d
+			y: -0.5d
+			description: [
+				"Industrial Foregoing has massive storage capabilities. "
+				""
+				"Head over to the Industrial Foregoing Chapter to learn about Black Hole Storage."
+			]
+			dependencies: ["568DC3ECDB943C47"]
+			id: "1A0E0EC91FB287B0"
+			tasks: [{
+				id: "74238F0F4B713CBE"
+				type: "item"
+				item: "industrialforegoing:common_black_hole_tank"
+			}]
+			rewards: [{
+				id: "05BBC26D4E6D383F"
+				type: "xp"
+				xp: 100
+			}]
+		}
+		{
+			title: "Fluid Tank (Create)"
+			x: 10.0d
+			y: -2.0d
+			description: ["Create tanks automatically merge with adjacent tanks and can be up to 3x3."]
+			dependencies: ["568DC3ECDB943C47"]
+			id: "516772B582C9114A"
+			tasks: [{
+				id: "755A21A9FE7980B4"
+				type: "item"
+				item: "create:fluid_tank"
+			}]
+			rewards: [{
+				id: "6A4A37F812BE7CA6"
+				type: "xp"
+				xp: 100
+			}]
+		}
+		{
+			title: "Ender Tank"
+			x: 10.0d
+			y: -3.0d
+			description: ["Ender Tanks will share their inventories with other tanks using the same color code. Try them for pipeless fluid transport."]

```

Comment:
I think you should mention that you can set the color code network to privat by clicking with a diamond.